control/tests/lti_test.py

```diff
@@ -70,6 +70,15 @@ def test_damp(self):
         np.testing.assert_almost_equal(sys_dt.damp(), expected_dt)
         np.testing.assert_almost_equal(damp(sys_dt), expected_dt)
 
+        #also check that for a discrete system with a negative real pole the damp function can extract wn and theta.
+        p2_zplane = -0.2
+        sys_dt2 = tf(1,[1,-p2_zplane],dt)
```

Comment:
Please add some spaces for PEP8 style compliance.

```suggestion
        sys_dt2 = tf(1, [1, -p2_zplane], dt)
```